src/Infrastructure/Identity/Services/EmployeeService.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System.Net.Http.Headers;
+using System.Net.Http.Json;
+using Microsoft.Extensions.Configuration;
+using MRA.AssetsManagement.Domain.Entities;
+
+namespace MRA.AssetsManagement.Infrastructure.Identity.Services;
+
+public class EmployeeService : IEmployeeService
+{
+    private readonly HttpClient _http;
+    private readonly IConfiguration _configuration;
+
+    public EmployeeService(HttpClient http,IConfiguration configuration)
+    {
+        _http = http;
+        _configuration = configuration;
+    }
+    public async Task<List<EmployeeResponse>> GetAll(string token)
+    {
+        
+        SetAuthorizationHeader(token);
+        var response = await _http.GetFromJsonAsync<List<EmployeeResponse>>("https://localhost:7245/api/User/GetListUsers/ByFilter");
```

Comment:
`https://localhost:7245/api/User/GetListUsers/ByFilter`

get Identity server address from the configuration file